lib/senders/templates/_versions.json

```diff
@@ -3,6 +3,7 @@
   "passwordChangedEmail": 1,
   "passwordResetEmail": 1,
   "passwordResetRequiredEmail": 1,
+  "postChangePrimaryEmail": 1,
```

Comment:
This time did it work??

Reply:
Script worked, but I ran it manually after adding templates.